[ntuple][skip-ci] update release notes for SetQuantized (#22136)
root-project/root: README/ReleaseNotes/v640/index.md

  Secure communication should be provided by standard means, such as SSL sockets or SSH tunneling.
* The `builtin_davix` build option has been removed.
  The Davix I/O code in ROOT remains uneffected and is built as before provided that the Davix library is found on the system.

> * `RRealField::SetQuantized` now has a new overload and the existing signature has been deprecated. The new overload prevents errors due to passing arguments in the wrong order.

Reviewer: To switch to a positive tone.
```suggestion
* `RRealField::SetQuantized` now has a new overload and the existing signature has been deprecated. The new overload enforces the proper ordering of the arguments.
```
